deps/tools/bb-install.mk

```diff
@@ -47,7 +47,12 @@ UNINSTALL_$(strip $1) := $$($(2)_JLL_BASENAME:.tar.gz=) bb-uninstaller
 $$(build_prefix)/manifest/$(strip $1): $$(SRCCACHE)/$$($(2)_JLL_BASENAME) | $(build_prefix)/manifest
 	-+[ ! -e $$@ ] || $$(MAKE) uninstall-$(strip $1)
 	$$(JLCHECKSUM) $$<
-	mkdir -p $$(build_prefix)
+ifneq (bsdtar,$(findstring bsdtar,$(TAR_TEST)))
+	@# work-around a gtar bug: they do some complicated work to avoid the mkdir
+	@# syscall, which is buggy when working with Tar.jl files so we manually do
+	@# the mkdir calls first in a pre-pass
+	$(TAR) -tzf $$< | xargs dirname | sort -u | (cd $$(build_prefix) && xargs -t mkdir -p)
```

Comment:
I don't think you can pass multiple arguments to `dirname`

Reply:
Why is `gtar` even on the path on the macOS bots?

Reply:
Of course it's only on the very last buildbot that I check for.  :P

Someone installed it two years ago, presumably for some purpose.  It has been removed.